packages/ui-extensions/src/surfaces/customer-account/extension.ts

```diff
@@ -19,3 +19,7 @@ export const extension =
  * @deprecated This is deprecated, use `extension` instead.
  */
 export const extend = extension;
+
+export interface RunExtension<Api, Return> {
+  (api: Api): Promise<Return>;
+}
```

Comment:
from here https://github.com/Shopify/ui-extensions/blob/main/packages/customer-account-ui-extensions/src/extension-points/extension-signature.ts 

only for return, should be removed soon

Reply:
Sure. Will remove as well.